config/stripe/subscription.js

```diff
@@ -1,7 +1,7 @@
 export default {
-  enabled: true,
```

Comment:
How these changes relate to error handling?

Reply:
Module payment has been changed within this issue and to start this module I should have enabled this module, and after making changes disable.

Reply:
fix